src/README.md

```diff
@@ -40,7 +40,7 @@ test-configuration/
 │   ├── exportS3ProductionBucketName.sh     
 │   └── exportS3StagingBucketName.sh        
 └── style                                   
-    ├── _overides.scss                      # contains main theme colors
+    ├── _overrides.scss                      # contains main theme colors
```

Comment:
This document seems inaccurate.  I guess I expected the test-configuration to have the same changes as other portals configurations (delete _overides.scss, and create _style_overrides.scss and _variable_overrides).

Reply:
README could be expanded to explain the scss compilation strategy.

Reply:
Yup--I updated this once, made more changes, and then forgot to update it again. Will fix + add content to describe the whole process